src/service.cc

```diff
@@ -420,6 +415,27 @@ ServiceLayer::stream(const std::string hashtag, std::time_t seconds,
   }
   return to_send;
 }
+std::vector<chirp::Chirp> ServiceLayer::TestStream(const std::string hashtag) {
+  std::time_t seconds;
+  int64_t microseconds_since_epoch;
+  chirp::StreamReply reply;
+  chirp::Chirp chirp;
+  std::vector<chirp::Chirp> to_send_back;
+  int number_of_times_to_check_chirps = 10;
```

Comment:
Minor: rename this to something that describes the variable as it changes, not just the starting condition.  That is, something like `chirp_checks_left`.